packages/headers-parser/src/all.ts

```diff
@@ -11,6 +13,11 @@ export const parseAllHeaders = async function ({
   netlifyConfigPath,
   configHeaders = [],
   minimal = false,
+}: {
+  headersFiles?: undefined | string[]
+  netlifyConfigPath?: undefined | string
+  configHeaders: undefined | MinimalHeader[]
+  minimal: undefined | boolean
```

Comment:
should these two be optional?

Reply:
Thanks, I took a closer look here and fully refined them to reflect reality. `minimal` actually can be required, so I'll cut a major.